README.md

```diff
@@ -26,8 +35,9 @@ The LoRaWAN starter kit is an *experimental* cross platform private network impl
 - Easy deployment and setup using Azure ARM templates.
 - Small to Midsize Scalability Tests.
 - Simulator for development and testing without the need to own a Gateway.
-  
-## LoRaWAN Specification Support
```

Comment:
I know that you did not edit this, this time around, but let's chance the spacing of the "LoRaWAN Specification Support" to be consistant with the "Features" section. Currently after every bulletted point, there is a blank line --> let's remove those.